isis/tests/FunctionalTestsOverlapstats.cpp

```diff
@@ -0,0 +1,199 @@
+#include <QTextStream>
+#include <QStringList>
+#include <QTemporaryFile>
+
+#include "findimageoverlaps.h"
+#include "overlapstats.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/overlapstats.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestOverlapstatsBadCubeList) {
+  QString badCubeList = tempDir.path() + "/badcubes.lis";
+  QString overlapsPath = tempDir.path() + "/overlaps.lis";
+  FileList cubes;
+
+  // badCubeList only contains cube1, overlaps has cube1 and cube2
+  cubes.append(FileName(cube1->fileName()));
+  cubes.write(badCubeList);
+  cubes.append(FileName(cube2->fileName()));
+  cube1->close();
+  cube2->close();
+
+  QString FIO_APP_XML = FileName("$ISISROOT/bin/xml/findimageoverlaps.xml").expanded();
+  QVector<QString> fio_args = { "OVERLAPLIST=" + overlapsPath };
+  UserInterface fioOptions(FIO_APP_XML, fio_args);
+  findimageoverlaps(cubes, fioOptions, false, nullptr);
```

Comment:
These calls to findimageoverlaps need to be replaced using the overlap.lis file that findimageoverlaps produces. I'm fine with creating the overlap files then adding them to the code base. Like @Kelvinrr mentioned before we don't want these gtest app tests to be dependent on other apps.

Reply:
In the name of keeping momentum, we agreed that smaller issues like this can be punted and dealt with later when the size of the data area is smaller. Since Tim has had issues costing him a few days already (trying to get a overlap file working included), I think it's best to punt on the problem for now.